Audit log throws unexpected error on start of the service

**Describe the bug**
Any service using the ARC Audit Service fails at start with the following error when using it after docker prune -

```
Starting newrelic agent
60
node:internal/modules/cjs/loader:1143
59
  throw err;
58
  ^
57
56
Error: Cannot find module './doc/workbook'
55
Require stack:
54
- /home/node/app/node_modules/exceljs/lib/exceljs.nodejs.js
53
- /home/node/app/node_modules/exceljs/excel.js
52
- /home/node/app/node_modules/@sourceloop/audit-service/dist/services/audit-log-export.service.js
51
```


**To Reproduce**
Steps to reproduce the behavior:
1. install the latest ARC Audit Service
2. bind it's component in any Loopback application
3. Run [node-prune](https://github.com/tj/node-prune)
4. Start the service
5. See error

**Expected behavior**
The service should start without any error


**Additional context**
The issue seems to be because of node-prune removing `docs` named folders from node_modules, and excejs used by the latest audit service seems to be dependent on it

--- a/services/audit-service/README.md
+++ b/services/audit-service/README.md
@@ -219,7 +219,7 @@ If `includeArchivedLogs` option is set to `false` (which is default if not provi
 ### Export Logs
 
 This feature is used to export the logs present in Audit Database or the archive storage(eg. AWS.S3). A default loopback filter is accepted based on which logs are exported to the desired location as specified as an excel file(by default) using `AuditLogExportProvider` [here](src/providers/audit-log-export.service.ts).
-The exceljs dependency, used by the `AuditLogExportProvider` for generating Excel files, is optional. Users who wish to utilize this feature should manually install exceljs and bind the AuditLogExportProvider in the application.ts file of your application:
+The exceljs dependency, used by the `AuditLogExportProvider` for generating Excel files, is optional. Users who wish to utilize this feature should manually install exceljs and bind the AuditLogExportProvider in the application.ts of your application:
 
 ```ts
 this.bind(AuditLogExportServiceBindings.EXPORT_AUDIT_LOGS).toProvider(

--- a/services/audit-service/package.json
+++ b/services/audit-service/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@sourceloop/audit-service",
   "version": "13.0.0",
-  "description": "Audit logging microservice",
+  "description": "Audit logging Microservice.",
   "keywords": [
     "loopback-extension",
     "loopback"

